submissions/BukkyOyetimehin/domManipulation/signUPForm.js

```diff
@@ -0,0 +1,151 @@
+let signupForm = document.querySelector('#validateForm'),
+    firstName = document.querySelector('#fname'),
+    lastName = document.querySelector('#lname'),
+    otherNames = document.querySelector('#onames'),
+    phoneNumber = document.querySelector('#phone'),
+    emailAddress = document.querySelector('#email'),
+    password = document.querySelector('#pwd'),
+    confirmPassword = document.querySelector('#cpwd'),
+    error = document.querySelector('.error-block');
+//First of all, the form must be checked to make sure all the mandatory fields are filled in.
+function required(field, event) {
+    if (fname.value === '') {
+        fname.nextElementSibling.innerHTML = "Please enter your First Name.";
+        event.preventDefault();
+        return false;
+    }
+    if (lname.value === '') {
+        lname.nextElementSibling.innerHTML = "Please enter your Last Name.";
+        event.preventDefault();
+        return false;
+    }
+    if (phone.value === '') {
+        phone.nextElementSibling.innerHTML = "Please enter your Phone Number.";
+        event.preventDefault();
+        return false;
+    }
+    if (email.value === '') {
+        email.nextElementSibling.innerHTML = "Please enter your Email Address.";
+        event.preventDefault();
+        return false;
+    }
+    if (pwd.value === '') {
+        pwd.nextElementSibling.innerHTML = "Please enter your Password"
+        event.preventDefault();
+        return false;
+    }
+    if (cpwd.value === '') {
+        cpwd.nextElementSibling.innerHTML = "Please confirm your Password.";
+        event.preventDefault();
+        return false;
+    }
+    field.nextElementSibling.innerHTML = "";
+    return true;
+}
+
+//Validate Form
+function validateForm(e) {
+    required(firstName, e);
+    required(lastName, e);
+    required(phoneNumber, e);
+    required(email, e);
+    required(password, e);
+    required(confirmPassword, e);
+    firstNameFieldValidation(e);
+    lastNameFieldValidation(e);
+    otherNamesFieldValidation(e);
+    phoneNumberFieldValidation(e);
+    emailFieldValidation(e);
+    passwordValidation(e);
+    confirmPasswordValidation(e)
+    return true;
+}
+signupForm.addEventListener("submit", validateForm);
+//Secondly, the data that is entered must be checked for correct form and value.
+//Validate First Name
+function firstNameFieldValidation(event) {
```

Comment:
You are repeating your self too much, these functions are all doing the same thing, create a single function and pass all other things that make them differ as a parameter